code/game/objects/items.dm

```diff
@@ -53,6 +52,10 @@
 	var/vending_cat = null// subcategory for vending machines.
 	var/list/dynamic_overlay[0] //For items which need to slightly alter their on-mob appearance while being worn.
 
+	var/is_shrapnel = 0
+	var/shrapnel_amount = 0
+	var/shrapnel_type = ""
```

Comment:
This is the way it is done for guns and ammo though, which is what I was looking at when I wrote (Frankensteined) this code. If I remove the qoutes, will I need to change this (from the prime() of the bomb) aswell?
var/obj/item/projectile/bullet/shrapnel_projectile = new shrapnel.shrapnel_type(src)